### DISCUSSION TITLE
skip/hide unsupported or not required fields in case class

### DISCUSSION THREAD

**Discussion by JerinThomas25** [andyglow/scala-jsonschema#214] [Q&A] [answered]
@andyglow 
Some queries regarding this project

1. Is there any way to skip/hide some unsupported fields in one case class
ie `case class test(value: ZonedDateTime)`  //ZonedDateTime not supported

2. is it possible to add custom Json Schema for unsupported types.

**Comment by andyglow** [comment, marked as answer]
Hello @JerinThomas25 
1. for skip/hide - there are no such a functionality implemented
2. yes, it's possible, please find examples here https://github.com/andyglow/scala-jsonschema/blob/master/modules/joda-time/src/main/scala/com/github/andyglow/jsonschema/JodaTimeSupport.scala
